tutorial.md

```diff
@@ -0,0 +1,126 @@
+# Deploying Moodle on Acorn
+
+[Moodle™ LMS](https://moodle.org/) is an open source online Learning Management System widely used at universities, schools, and corporations. It is modular and highly adaptable to any type of online learning.
+
+[Acorn](http://www.acorn.io) is a cloud computing platform with a big free sandbox that anyone can use by registering with a GitHub account. It is designed to simplify running modern  cloud-native apps on the public cloud. You use familiar development and deployment workflows based on mainstream container tools without having to deal with provisioning or configuring any underlying cloud resources. Basically it provides all the power of Kubernetes and Terraform, without any of the complexity.
```

Comment:
Instead of the templated style of what is and so on. We can start with an introduction describing why people go for Moodle and how deploying it on Acorn benefits users.